src/Appwrite/Template/Template.php

```diff
@@ -125,4 +176,36 @@ public static function fromCamelCaseToDash($input): string
     {
         return \str_replace([' ', '_'], '-', \strtolower(\preg_replace('/([a-zA-Z])(?=[A-Z])/', '$1-', $input)));
     }
+
+    /**
+     * setHtml
+     *
+     * Set the Raw html used by this template  
+     *
+     * @return string
+     * 
+     */
+    public function setHtml(string $html): self 
```

Comment:
Don't see a reason to have a dedicated method for this. We can just set the content directly in the  `fromString` method

Reply:
Yes but I've defined `fromString` to be a static method. `setHtml()` would act on an instance. Let me know if this is not required.